www/src/pages/components/alerts.mdx

```diff
@@ -47,12 +47,16 @@ Alerts can contain whatever content you like. Headers, paragraphs, dividers, go
 
 ### Dismissing
 
-Add the `dismissible` prop to add a functioning dismiss
-button to the Alert.
+Add the `dismissible` prop to add a dismiss button to the Alert.
 
-<ReactPlayground codeText={Dismissible} />
```

Comment:
Seems like we already have an example for that, let's eather keep it or enhance it since that would end up in an unused file.